waku/v2/node/wakunode2.nim

```diff
@@ -34,15 +35,15 @@ import
   ./wakunode2_types
 
 export
-  wakunode2_types
+  wakunode2_types,
+  peer_manager
```

Comment:
> Another quick fix would be importing peer manager in chat2 and wakubridge.

This is what I suggest, exactly. Let's make imports explicit rather than implicit.